paper.md

```diff
@@ -0,0 +1,74 @@
+  orcid: 0000-0002-2880-1243
+  affiliation: "2, 3"
+- name: Hamish Beath
+  orcid: 0000-0002-5124-9143
+  affiliation: "1, 2"
+- name: Jenny Nelson
+  orcid: 0000-0003-1048-1330
+  affiliation: "1, 2"
+affiliations:
+ - name: Department of Physics, Blackett Laboratory, Imperial College London, SW7 2AZ, United Kingdom
+   index: 1
+ - name: Grantham Institute - Climate Change and the Environment, Imperial College London, SW7 2AZ, United Kingdom
+   index: 2
+ - name: Department of Chemical Engineering, Imperial College London, SW7 2AZ, United Kingdom
+   index: 3
+date: 19 August 2022
+bibliography: paper.bib
+---
+
+# Summary
+
+Sustainable Development Goal 7 aims to provide sustainable, affordable, reliable and modern energy access to all by 2030. In order for this goal to be achieved, sustainable energy interventions in developing countries must be supported with design tools which can evaluate the technical performance of energy systems as well as their economic and environmental impacts.
+
+CLOVER (Continuous Lifetime Optimisation of Variable Electricity Resources) is a software tool for simulating and optimising community-scale energy systems, typically minigrids, to support energy access in developing countries. CLOVER can be used to model electricity demand and supply at an hourly resolution, for example allowing users to investigate how an electricity system might perform at a given location. CLOVER can also identify an optimally-sized energy system to meet the needs of the community under specified constraints, for example a system which provides a desired level of reliability at the lowest cost of electricity. CLOVER can provide an insight into the technical performance, costs, and environmental impact of a system, and allow the user to evaluate many different scenarios to decide on the best way to provide sustainable, affordable and reliable electricity to a community.
+
+CLOVER can be used on both personal computers and high-performance computing facilities. Its design separates its general framework (code) from user inputs (data) specific to their investigations, which are input via `YAML` files. Its straightforward command-line interface provides simple operation for both experienced Python users and those with little prior exposure to coding.
+
+# Statement of need
+
+CLOVER was developed to provide a robust techno-economic and environmental analysis tool to support academics, practitioners and policymakers in evaluating sustainable energy interventions in developing countries. By providing an open-source alternative to proprietary minigrid design tools such as HOMER [@HOMER], CLOVER offers a more comprehensive analysis tool than common spreadsheet-based alternatives. CLOVER was designed to provide a fast, flexible and user-friendly software to assess energy systems at both short-term (hourly) and multi-year timescales under a wide variety of user-defined scenarios, constraints and optimisation conditions.
```

Comment:
:bug: I think we're writing citations wrong. 

From the JOSS website:
> # Citations

> Citations to entries in paper.bib should be in
[rMarkdown](http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html)
format.

> If you want to cite a software repository URL (e.g. something on GitHub without a preferred
citation) then you can do it with the example BibTeX entry below for @fidgit.

> For a quick reference, the following citation commands can be used:
> - `@author:2001`  ->  "Author et al. (2001)"
> - `[@author:2001]` -> "(Author et al., 2001)"
> - `[@author1:2001; @author2:2001]` -> "(Author1 et al., 2001; Author2 et al., 2002)"

The way we've got "HOMER [@HOMER]" I think is not going to work. Rather, I think we need `[@HOMER]` with backticks "`" included.

Reply:
Thanks for spotting this - I've updated all of the instances of this